tests/test_phptravels.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+import time
+
+class ByLocators:
+    TAG_NAME = By.TAG_NAME, "tag name from the project to find element by"
```

Comment:
for all this locators you should use real path(just pick any element from page, it also could be different elements)

Reply:
ok, it is done